CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 
 ## [Unreleased](https://github.com/idealista/prometheus_consul_exporter-role/tree/develop)
-
+### Added
+- Updagrading consul_exporter from [0.3.0](https://github.com/prometheus/consul_exporter/releases/tag/v0.3.0) to [0.7.1](https://github.com/prometheus/consul_exporter/releases/tag/v0.7.1)
```

Comment:
You should tag the issue #1 at the begining of the description for an easier tracking in the future.

Reply:
Also, upgrading a version probably is better placed under "Changed" category rather than "Added". And please mention that this is a breaking change since you cannot run this role version with consul-exporter v0.3.0

Reply:
Done